backend/server/routes/index.js

```diff
@@ -1,14 +1,32 @@
-import { Router } from 'express';
-const router = Router();
+import { Router } from 'express'
 
-import { user, test } from '../controllers';
+import { user, posts, test } from '../controllers'
 
-router.route('/users').post(user.addUser);
+const router = Router()
 
-router.route('/users/:id').get(user.getUsersById);
+/*
+USERS
+*/
+router.route('/users').post(user.addUser)
 
-router.route('/test').get(test.helloWorld);
+router.route('/users/:id').get(user.getUsersById)
 
-router.route('/users/authenticate').post(user.authenticateUser);
+router.route('/users/authenticate').post(user.authenticateUser)
 
-export default router;
+/*
+POSTS
+*/
+router.route('/posts').post(posts.createPost)
+
+router.route('/posts/:id').get(posts.getPostById)
+
+router.route('/posts/:id').put(posts.modifyPostById)
+
+router.route('/posts/:id').delete(posts.deletePostById)
```

Comment:
You could do 
`router.route('/posts/:id').get(posts.getPostById).put(posts.modifyPostById).delete(posts.deletePostById)` since they all have the same url

Reply:
Fixed this in [Cleaned up routes](https://github.com/SE701Team2/Updog/pull/38/commits/f8480556fada553aca484cff47a40350fdbc0a15)
[f848055](https://github.com/SE701Team2/Updog/pull/38/commits/f8480556fada553aca484cff47a40350fdbc0a15).